pbconf/fossology.pb

```diff
@@ -46,14 +51,19 @@ sshport fossology = 22
 projver fossology = 3.1.0
 projtag fossology = 1
 
+#
 # Is it a test version or a production version
+#
 testver fossology = true
 
+#
 # Additional repository to add at build time
-# addrepo centos-5-x86_64 = http://packages.sw.be/rpmforge-release/rpmforge-release-0.3.6-1.el5.rf.x86_64.rpm,ftp://ftp.project-builder.org/test/centos/5/pb.repo
-# addrepo centos-4-x86_64 = http://packages.sw.be/rpmforge-release/rpmforge-release-0.3.6-1.el4.rf.x86_64.rpm,ftp://ftp.project-builder.org/test/centos/4/pb.repo
-addrepo centos-7-x86_64 = http://ftp.project-builder.org/centos/7/x86_64/pb.repo,http://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-9.noarch.rpm
+#
+#addrepo centos-5-x86_64 = http://packages.sw.be/rpmforge-release/rpmforge-release-0.3.6-1.el5.rf.x86_64.rpm,ftp://ftp.project-builder.org/test/centos/5/pb.repo
+#addrepo centos-4-x86_64 = http://packages.sw.be/rpmforge-release/rpmforge-release-0.3.6-1.el4.rf.x86_64.rpm,ftp://ftp.project-builder.org/test/centos/4/pb.repo
+addrepo centos-7-x86_64 = http://ftp.project-builder.org/centos/7/x86_64/pb.repo,https://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-10.noarch.rpm
```

Comment:
The link `https://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-10.noarch.rpm` returns 404

Reply:
Seems Fedora updated again their rpm:
https://dl.fedoraproject.org/pub/epel/7/x86_64/Packages/e/epel-release-7-11.noarch.rpm

Normally the latest version should be used:
https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm

But I think we changed as we had an issue using this in the past (even if I don't remember why :-()

Reply:
If we want reproducible builds we have to pin versions.

Reply:
Well, anyway we already use the generic version in the rest of the Vagrant setup for now. And that has no real impact on reproduceable builds (which we will never have anyway with composer downloaded on the fly, or packages installed from remote sources we don't control, so teir version may evolve). However not pinning makes maintenance much more easy.